### PR TITLE
Use gulp-load-plugins.

### LINKED ISSUE #43: gulp-load-plugins
[do it](https://github.com/jackfranklin/gulp-load-plugins)

### PR DESCRIPTION
**Pull request by jmeas**
Resolves #43

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -38,34 +27,34 @@ function ding(file) {
...
-    .pipe(preprocess())
-    .pipe(rename(config.exportFileName + '.js'))
-    .pipe(sourcemaps.init())
-    .pipe(to5({blacklist: ['useStrict'], modules: 'ignore'}))
-    .pipe(sourcemaps.write('./'))
+    .pipe($.template(config))
+    .pipe($.preprocess())
+    .pipe($.rename(config.exportFileName + '.js'))
+    .pipe($.sourcemaps.init())
+    .pipe($['6to5']({blacklist: ['useStrict'], modules: 'ignore'}))
```
Comment (jmeas) [inline gulpfile.js]: lol

Comment (jmeas) [inline gulpfile.js]: [oh nvm](https://github.com/jackfranklin/gulp-load-plugins#options).

File: gulpfile.js [hunk 2]
```diff
@@ -1,20 +1,11 @@
+var gulp = require('gulp');
+var $ = require('gulp-load-plugins')({
+  replaceString: /^gulp(-|\.)([0-9]+)?/
```
Comment (jmeas) [inline gulpfile.js]: Convert `gulp-6to5` to be `to5`

Comment (thejameskyle) [inline gulpfile.js]: lol... kill me.

Comment (jmeas) [inline gulpfile.js]: lol

Comment (jmeas) [inline gulpfile.js]: the alternative was `$['6to5']

Comment (jmeas) [inline gulpfile.js]: i might also do `$.to5 = require('gulp-6to5')` manually.

**Pull request merged**